openstack_lease_it/lease_it/backend/TestConnection.py

```diff
@@ -36,26 +40,29 @@ def _instances(self):
 
         :return: dict()
         """
-        response = cache.get('instances')
+        if GLOBAL_CONFIG['RESET_CACHE_INSTANCES']:
```

Comment:
Remplacer par
if not GLOBAL_CONFIG['RESET_CACHE_INSTANCES']:
    response = cache.get('instances')